pipeline/metadata/6.0.yaml

```diff
@@ -440,6 +440,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "tier-2_rgw_test-s3cmd-malformed-url-with-different-commands"
+          execution_time: "2h 49m 32s"
+          suite: "suites/pacific/rgw/tier-2_rgw_test-s3cmd-malformed-url.yaml"
+          global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
+          platform: "rhel-8"
```

Comment:
please change platform and inventory to rhel-9

Reply:
thank you @ckulal , please review again